### ISSUE TITLE
Call to strncat_s missing arg?

### ISSUE THREAD

**Issue by woollybah** [SLikeSoft/SLikeNet#40]
Platform : Win32, x64, MinGW 7.2

You seem to be missing a dest size argument here :
https://github.com/SLikeSoft/SLikeNet/blob/70a86d11b7595b6ac64bdd261ce440ff35ad4a64/Source/src/TelnetTransport.cpp#L96

**Comment by Luke1410** [comment]
Not quite. text is a char array with a fixed size (REMOTE_MAX_TEXT_INPUT) (ln 75). Therefore, the templated version of strncat_s() is used and the size of the destination buffer doesn't need to be specified as another argument by the caller.

Please note that we don't test explicitly on MinGW atm. Adding support is on our mid-term roadmap (internal case number: SLNET-237). Do you suggest that the code line there triggers a compile error atm with MinGW?
